recipes/gtest/all/patches/gtest-1.10.0-override.patch

```diff
@@ -0,0 +1,450 @@
+From 3cddd56e195b516f449bea6dcd3edd4494195631 Mon Sep 17 00:00:00 2001
+From: Robert Luberda <robert@debian.org>
+Date: Wed, 9 Oct 2019 21:48:00 +0200
+Subject: [PATCH] Add more override keywords
+
+Mark more functions with "override" keyword, just like
+it was done in commit 2460f97152c.
+
+This should prevent compiler from complaining while compiling both
+user code, and the googletest code itself with the -Wsuggest-override
+option turned on; with the exception of:
+ * calls to new MOCK_METHOD() in test/gmock-function-mocker_test.cc
+ * calls to old MOCK_METHODx()/MOCK_CONST_METHODx() in other
+   unit test files.
+
+Closes #2493
+---
+ .../include/gmock/gmock-generated-actions.h   | 24 ++---
+ .../gmock/gmock-generated-actions.h.pump      |  4 +-
+ .../include/gmock/gmock-generated-matchers.h  | 88 +++++++++----------
+ .../gmock/gmock-generated-matchers.h.pump     |  8 +-
+ googletest/include/gtest/gtest-typed-test.h   |  4 +-
+ .../include/gtest/internal/gtest-port.h       |  4 +-
+ googletest/test/gtest_unittest.cc             |  4 +-
+ 7 files changed, 68 insertions(+), 68 deletions(-)
+
+diff --git a/googlemock/include/gmock/gmock-generated-actions.h b/googlemock/include/gmock/gmock-generated-actions.h
+index 981af78ff..cee96dae8 100644
+--- a/googlemock/include/gmock/gmock-generated-actions.h
++++ b/googlemock/include/gmock/gmock-generated-actions.h
+@@ -663,7 +663,7 @@ class ActionHelper {
+       typedef typename ::testing::internal::Function<F>::ArgumentTuple\
+           args_type;\
+       explicit gmock_Impl GMOCK_INTERNAL_INIT_##value_params {}\
+-      virtual return_type Perform(const args_type& args) {\
++      return_type Perform(const args_type& args) override {\
```

Comment:
[override](https://en.cppreference.com/w/cpp/language/override) keyword is C++11 and later.
but should be safe as GTest 1.10 requires C++11 compliant compiler.

Reply:
The `override` keyword is only missing on some macros (the ones patched here) and it is present it the rest, if needed.
For reference you can see how the new MOCK_METHOD introduced on 1.10 works, it will add `override` to overridden methods.
So the patch is safe.